### ISSUE TITLE
"define_connection_info is %t, no further action"

### ISSUE THREAD

**Issue by fopina** [Telmate/terraform-provider-proxmox#1019]
https://github.com/Telmate/terraform-provider-proxmox/blob/be2585614955f7ada0433c9a1fe34e604bb717cd/proxmox/resource_vm_qemu.go#L1837

Is it really intentional to ALWAYS display this warning whenever it is set to false?

Not only `%t` doesn't get any value but also: I know I set it to false, I'd like to not have the warning popup every time...

**Comment by Tinyblargon** [comment]
@fopina any reason you are setting it to false?
As far as i understand it `define_connection_info` doesn't do anything except returning that error and not asking the guest agent for information.

https://github.com/Telmate/terraform-provider-proxmox/blob/0db272002ff16db4201402fbb9bc0ddc4161ea41/proxmox/resource_vm_qemu.go#L1898-L1903

**Comment by fopina** [comment]
@Tinyblargon exactly, I don't install the agent so I didn't want it to query (and fail)
